cluster/addons/fluentd-elasticsearch/fluentd-es-image/Dockerfile

```diff
@@ -19,8 +19,10 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM gcr.io/google-containers/debian-base-amd64:0.1
+FROM debian:stretch-slim
 
+COPY clean-apt /usr/bin
```

Comment:
this file is being added now and I should have noticed it was missing from the prior version.  

- where does this get built and then pushed to the official repo?  
- can we add that as a unit test so we have a clear statement that 'the Dockerfile checked in will build'?